server/jobs/example.py

```diff
@@ -1,14 +1,27 @@
 import time
+import io
 
 from server import jobs
+from server.models import ExternalFile
+from server.utils import encode_id
 
 @jobs.background_job
-def test_job(duration=0, should_fail=False):
+def test_job(duration=0, should_fail=False, make_file=False):
     logger = jobs.get_job_logger()
 
     logger.info('Starting...')
     time.sleep(duration)
     if should_fail:
         1/0
+    if make_file:
+        data = io.BytesIO(b'print("Hello World")\n'*duration)
```

Comment:
Should we make this an iterable as an example on how to create a streaming iterator? I think using StringIO/BytesIO is a bit of an anti-pattern (and caused v2 to run out of memory). Something like

```
def data():
    for _ in range(duration):
        yield b'print("Hello World")\n'
```

Incidentally, what happens if we yield strings instead of bytes?